### DISCUSSION TITLE
How to return to cast screen on GoogleTV if user has exited it

### DISCUSSION THREAD

**Discussion by Kanuan** [Tapanila/SharpCaster#314] [General]
I have an app that does the following:

- Connect to a chromecast
- Use LoadMediaAsync to send a image (mediatype is set to image)

I'm testing with Chromecast with GoogleTV (CCwGTV) and am noticing the following behavior: if I start casting with SharpCaster then get out of the cast screen by using the "home" or "back" buttons...:

- Casting continues running into the background, meaning disconnection is NOT triggered
    - Attempting to SetMute() or SetVolume() will work normally
- GoogleTV does not offer any way of getting back to the casting screen
    - If I'm casting a video via the CrunchyRoll app, GoogleTV will show a mini-player in the home screen that allows me to get back to the casting screen. This mini-player does not appear when using SharpCaster (at least when casting images, didn't test with videos)
- Sending a new media of the type image does not make GoogleTV switch back to the Cast screen
- Attempting to re-locate chromecasts and then connecting again to the same one the app was already connected won't make the cast screen re-appear
- For the cast screen to reappear, the SharpCaster application casting needs to be first finalized on the GoogleTV side (restarting GoogleTV or triggering a disconnection from the SharpCaster side), then re-launch from "zero" the SharpCaster app

I'd like to know if there is something I'm missing that could be used to make GoogleTV switch back

**Comment by Tapanila** [comment]
Very interesting scenario. Can you try starting another application than the one you initially use and see if that does something?
